README.md

```diff
@@ -59,9 +59,10 @@ A collection of components to build a real time ingestion pipeline.
 
 ## Release Notes
 
-**1.2.0** PENDING
+**1.2.0**
 
 *   Upgrade to Kafka 2.0
+*   Added support for custom delimeter on the REDIS sink
```

Comment:
@sotirissl This didn't make it to the release so needs to be in the  1.3 Pending release notes